js/index.js

```diff
@@ -70,4 +68,29 @@ for (let i = 0; i < repositories.length; i++) {
 }
   projectList.appendChild(project);
 }
-})
\ No newline at end of file
+}) */
+
+//Fetch API
+fetch('https://api.github.com/users/Ljanjic/repos')
+	.then(response => {
+		return response.json();
+	})
+	.then(repositories => {
+		console.log(repositories);
+		//Display Repositories in List
+		let projectSection = document.getElementById("projects");
+		let projectList = projectSection.querySelector("ul");
+		for (let i = 0; i < repositories.length; i++) {
+			let project = document.createElement("li");
+			if (repositories[i].name === 'eridanus-intro') {
+				project.innerHTML = `<a href = "${repositories[i].html_url}">${repositories[i].name}</a>`
+			} else {
+				project.innerText = repositories[i].name;
+			}
+			projectList.appendChild(project);
+		}
+	})
+	.catch(error => {
+		console.log('You made some error!');
+		console.error(error);
```

Comment:
If you're just wanting to log an error to your console along with a message, you can combine line 94 and 95 by passing in additional arguments to console.error. For example:

`console.error('You made some error!', error)`

Additional Info: Depending on your philosophy around error handling, you can use `catch` to do a lot of really cool stuff. You can anticipate errors in the catch block to update the DOM to explain an error has happened. You can also return some default data and continue your promise chain, and more!

It can be a little complicated sometimes, but will be happy to explain this more if you're interested.